### ISSUE TITLE
[a11y] web app should have a meaningful title

### ISSUE THREAD

**Issue by chunhtai** [flutter/flutter#170715]
### Use case

https://www.w3.org/WAI/WCAG21/Understanding/page-titled

web needs a title widget to set the page title

### Proposal

add a guideline test to make sure the app has at least one title widget at all time
